local_flow/rec/src/utils.py

```diff
@@ -17,3 +21,58 @@ def return_json_file_content(file_name: str):
         data = json.load(json_file)
 
     return data
+
+class ExperimentTracker:
```

Comment:
We are defining the experiment tracker in two places in the project now. This is a potential source of bugs if we do changes on one and forget the other. It might be interesting to build a common package and share it across the different flows. This change is out of the scope of this PR but we should keep it in mind @jacopotagliabue

Reply:
Definitely! 

I caught that too, even the previous **local flow**`wandb` implementation was duplicated, as well as in the **remote flow**. It would require a significant change in the structure for example a global experiment tracker class for both local(intent/rec) and remote flow. I leave it for you to decide how you want to go about it.

Let me know if you need my help!

Reply:
Indeed. This is out of the scope of the PR. We could address such a change in another changeset if we decide to go that way. Will need Jacopo's input on this